pkg/process/checks/container.go

```diff
@@ -78,6 +78,9 @@ func (c *ContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Me
 		return nil, err
 	}
 
+	// Keep track of containers addresses
+	LocalResolver.LoadAddrs(ctrList)
```

Comment:
Have you sync'd with @mbakht on this at all?  I believe he is working on fixing the raciness of the container -> pid mapping and some structure like this might be needed.

Reply:
I'll sync with him once he's back 👍